### PR TITLE
Add report post about ELIXIR AHM 2023

### PR DESCRIPTION
**Pull request by bebatut**

### CODE REVIEW

File: content/news/2023-06-26-elixir-ahm/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+## Day 1 - June 5
+
+On the first day, Wolfgang gave the talk *"Galaxy - Galaxy workflows for taking viral genomic surveillance beyond SARS-CoV-2"*, in the [*Biodiversity, Food Security and Pathogens*](https://docs.google.com/document/d/1GOGaZdlLUD7gbg5HFlYWOmv4Nfppk1L0y2b3w1s-w1U/edit#heading=h.dqg3p1q9y270) mini-symposium.
+
+<twitter tweet="1665755049442648065"></twitter>
+<twitter tweet="1665752049231749120"></twitter>
+
+## Day 2 - June 6
+
+On the morning of 2nd day, Björn together with Nicola Soranzo ran the workshop [*Scaling user support: Galaxy from a service perspective*](https://docs.google.com/document/d/1kTot1V-t7xJx5ftQJiWiht5e3ix3nAq3MkFKhU4_4lA/edit#).
```
Comment (wm75) [inline content/news/2023-06-26-elixir-ahm/index.md:31]: At least this linked document is editable by everyone. Haven't checked the others, but that's likely not desirable.

File: content/news/2023-06-26-elixir-ahm/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+<twitter tweet="1666766756738736131"></twitter>
+
+All over the 4 days of the meeting, there was a **broad presence of Galaxy** across the different workshops, symposiums, plenary sessions, and posters. We will highlight in this post some of this presence.
+
+# Talks
+
+## Day 1 - June 5
+
+On the first day, Wolfgang gave the talk *"Galaxy - Galaxy workflows for taking viral genomic surveillance beyond SARS-CoV-2"*, in the [*Biodiversity, Food Security and Pathogens*](https://docs.google.com/document/d/1GOGaZdlLUD7gbg5HFlYWOmv4Nfppk1L0y2b3w1s-w1U/edit#heading=h.dqg3p1q9y270) mini-symposium.
```
Comment (wm75) [inline content/news/2023-06-26-elixir-ahm/index.md:24]:
```suggestion
On the first day, Wolfgang gave the talk [*Galaxy - Galaxy workflows for taking viral genomic surveillance beyond SARS-CoV-2*](https://docs.google.com/presentation/d/1l7eGLXa3Rah4uBHZdf2wtUs3jMMk0eox9CK2lb48Ck4/), in the [*Biodiversity, Food Security and Pathogens*](https://docs.google.com/document/d/1GOGaZdlLUD7gbg5HFlYWOmv4Nfppk1L0y2b3w1s-w1U/) mini-symposium.
```

Like the link to Pavan's presentation, this actually takes you to the full slide deck of the event, not just my slides, but probably ok like this?

**Review by bgruening — APPROVED** [review]
Awesome, thanks!

**Comment by bgruening** [comment]
Thanks a lot @bebatut!

**Pull request merged**